static/app/views/starfish/components/tableCells/spanDescriptionCell.tsx

```diff
@@ -33,7 +33,12 @@ export function SpanDescriptionCell({
       return rawDescription;
     }
 
-    return formatter.toSimpleMarkup(rawDescription);
+    return (
+      formatter
+        .toSimpleMarkup(rawDescription)
+        // Since the order of the tokens will never change, it's safe to assign the index as the key
+        .map((e, i) => cloneElement(e, {key: i}))
+    );
```

Comment:
🤔 can this code live in `sqlish/formatters/simpleMarkup.tsx` instead? That was every usage will have correct `key=` props

Reply:
I completely missed this file, thanks! I'll move the key logic here

Reply:
Moved it to `simpleMarkup.tsx`, how's it look now?

Reply:
Nice 👍🏻